releasenotes/notes/remove-cfun-toplevel-5d0d58bed83d57ed.yaml

```
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    Importing :class:`.Int1`, :class:`.Int2`, :class:`BooleanFunction`, :func:`classical_function`
+    from :mod:`qiskit.circuit` is now disabled. Instead, import the objects from the
+    :mod:`qiskit.circuit.classicalfunction` submodule.
```

```suggestion
    Importing :class:`.Int1`, :class:`.Int2`, :class:`.BooleanFunction`, :func:`.classical_function`
    from :mod:`qiskit.circuit` is now disabled. Instead, import the objects from the
    :mod:`qiskit.circuit.classicalfunction` submodule, which requires the ``tweedledum`` package.
```

or something like this, perhaps?